chpl/chpl-resources/src/main/resources/system-triggers.xml

```diff
@@ -169,5 +169,16 @@
                 <cron-expression>0 45 10 ? * 7#2</cron-expression> <!-- At 1045 UTC on the second Saturday of the month -->
             </cron>
         </trigger>
+        
+        <trigger>
+            <cron>
+                <name>listingValidationReportCreator</name>
+                <group>listingValidationReportCreatorTrigger</group>
+                <job-name>listingValidationReportCreator</job-name>
+                <job-group>systemJobs</job-group>
+                <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
+                <cron-expression>0 0 0 * * ?</cron-expression> <!-- At 0000 UTC every day -->
```

Comment:
We have a document in gitlab that has these scheduled jobs. I think this needs to be added to that document, and it wouldn't hurt to check that document to see that it's up to date

Reply:
I added that to my ToDo list before creating that PR.  Figured it could be done under the "GitLab Updated" checkbox.

Reply:
Right, it should be, but I needed to document the need for it somewhere so I wouldn't forget. Will close this conversation once I see that MR in place